### DISCUSSION TITLE
Supernote on iOS: share

### DISCUSSION THREAD

**Discussion by affvalente** [standardnotes/forum#2937] [Suggestions]
Option to share a note (for example a supernote) in a way that opens the default iOS share options, including share to email, messages, etc

The note text would then be properly formatted when composing the email or message.

The export function at the moment creates a file

**Comment by pouchofsilver** [comment]
I don't think StandardNotes is set up for sharing, supernote or otherwise. (Could be wrong). A byproduct of focusing on privacy and encryption.

**Comment by affvalente** [comment]
It’s not share in the sense of collaborate, where I agree with you SN is not built for that. 

This is share as in “send to another app within iOS while preserving the nice formatting because I, the owner of the note, want to send it”.

**Comment by pouchofsilver** [comment]
Right, but as you mentioned, those other apps include email and messages. Even though you are the owner/author,I think that is purposely locked down.

**Comment by affvalente** [comment]
Don't think so, otherwise they would also lock down "export" (create a file with the markdown of the note)

**Comment by myreli** [comment]
Currently, you can share on most editors, except for Super:

<img width="232" alt="image" src="https://user-images.githubusercontent.com/17554234/221393575-5e44cbb1-d18d-43e9-b247-0d600ceedd05.png">
